docs/dev-guide/contributing/merging.rst

```diff
@@ -119,73 +106,33 @@ website.
 - If your commit is on ``X.Y-dev``, you want to merge ``X.Y-dev`` to  the next
   higher version's dev branch. This will typically be ``X.Y+1-dev``.
 - When you are done merging through the various dev branches, merge the highest
   dev branch to ``master``.
 
-For example, if you commit to ``2.5-testing``, you would merge ``2.5-testing``
-into ``2.5-dev``, then ``2.5-dev`` into ``2.6-dev``, then ``2.6-dev`` into
-``master``. The trick here is that we never merge between testing or release
-branches; code only goes into dev branches and then through the dev
-branches into master.
-
-The following diagram that may be illustrative:
-
-.. image:: images/pulp-merging.png
-
-The arrows indiciate which direction the merges occur.
-
-.. _merging-to-multiple-releases:
+For example, if you merge a PR to ``2.7-dev`` and the only newer release branch
+is ``2.8-dev``, you would merge ``2.7-dev`` into ``2.8-dev``, then ``2.8-dev``
+into ``master``.
 
 You will also want to set the "Target Release" field in the redmine issue.
 "Target Release" is available for Issues, Refactors, and Stories but not for
 Tasks. If you have a task that appears to need a target release, please
 consider using one of the other three issue types.
 
-Merging to Multiple Releases
-----------------------------
-
-This advice applies to scenarios when you need to apply a fix to multiple
-releases. For example, you may need to fix something in both Pulp 2.5 and 2.6.
-
-The most important aspect of merging a change into multiple release branches is
-:ref:`choosing the right branch to start from <choosing-upstream-branch>`.
-
-Once your work is complete, submit a pull request from your GitHub fork into the
-branch for the oldest release you intend to merge into. Once review and revision
-is complete, merge your branch from the pull request web page. Do not delete the
-branch yet.
-
-At this point, you will do a "typical" forward merge, as documented :ref:`above
-<merging-your-changes>`.
-
-You will then want to take your branch with the PR, and merge that to the next
-branch that needs it. For example, if you have a fix you want in 2.5 and 2.6,
-you'd merge to ``2.5-testing`` and merge forward as usual, then merge your PR
-branch to ``2.6-testing`` and merge that forward as well. This will land your
-fix in both places cleanly.
-
-For cases where there are few merge conflicts, merge your working branch manually
-into each successively newer release branch, and finally into master. Generally,
-unless you are resolving conflicts or otherwise modifying your initial fix to
-accommodate the newer branches, no additional pull requests or review are needed.
-
 For cases where there are substantial merge conflicts whose resolution merits
-review, create a new branch from your working branch and merge the release branch
-into it. For example, assume you have branch "username-foo" from the "pulp-2.0"
-branch.
+review, create a new branch from your working branch and merge the release
```

Comment:
Similar question about the usage of "release branch" as stated [here](https://github.com/pulp/pulp/pull/2122/files#r42310585).